Makefile

```diff
@@ -519,9 +519,14 @@ electron-develop:
 		-t electron \
 		-s "$(PLATFORM)"
 
+babel:
+	$(NPX) babel --copy-files --out-dir lib lib && $(NPX) babel --copy-files --out-dir tests tests
+
 sass:
 	$(NPX) node-sass lib/gui/scss/main.scss > lib/gui/css/main.css
 
+transpile: babel sass
```

Comment:
You should also add these `babel` and `transpile` targets to the `TARGETS` list.